xml/System.Net.Security/SslApplicationProtocol.xml

```diff
@@ -374,9 +385,11 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Overrides the <see cref="M:System.Object.ToString" /> method.</summary>
+        <returns>Returns a string representation of this <see cref="T:System.Net.Security.SslApplicationProtocol" /> object.</returns>
+        <remarks>
+          If the <xref:System.Net.Security.SslApplicationProtocol.Protocol> contains a valid UTF8 bytes it will retrun text representation othervise hex representation of the value.
```

Comment:
This should take care of the build error you got. I also fixed a few typos. I didn't check the rest of the doc.
```suggestion
          <format type="text/markdown"><![CDATA[  

## Remarks  

If the <xref:System.Net.Security.SslApplicationProtocol.Protocol> contains valid UTF-8 bytes it will return text representation; otherwise, hex representation of the value.

 ]]></format>
```

Reply:
Is it mandatory to use markdown in the remarks? Is the purpose to render ##Remarks header?

Reply:
Not mandatory, just easier. Otherwise, you need to replace the xref links you had with the see cref syntax.